pkg/setting/setting.go

```diff
@@ -980,7 +984,24 @@ func NewCfg() *Cfg {
 		Logger: log.New("settings"),
 		Raw:    ini.Empty(),
 		Azure:  &azsettings.AzureSettings{},
+
+		// Avoid nil pointer
+		IsFeatureToggleEnabled: func(_ string) bool {
+			return false
+		},
+	}
+}
+
+func NewCfgWithFeatures(features FeatureToggles) *Cfg {
```

Comment:
Can you move this functionality to `NewCfg`? It would make this package easier to use.

Reply:
Got it, thanks :+1: